src/opt.rs

```diff
@@ -19,3 +24,20 @@ pub enum ChecksumVerificationMode {
     // on SSTable opening and on every block read.
     OnTableAndBlockRead,
 }
+
+impl Default for ChecksumVerificationMode {
+    fn default() -> Self {
+        Self::NoVerification
+    }
+}
+
+pub fn build_table_options(opt: &crate::AgateOptions) -> TableOptions {
+    // get latest data key
+    TableOptions {
+        table_size: opt.base_table_size,
+        block_size: opt.block_size,
+        bloom_false_positive: opt.bloom_false_positive,
+        checksum_mode: opt.checksum_mode,
+        table_capacity: (opt.base_level_size as f64 * 0.95) as u64,
```

Comment:
Why not `table_size` but `base_level_size`?

Reply:
Sorry, but what `table_size`? I think there is no problem we set `table_size` to `base_table_size` here. We will either change or not use this field later.